source/frontend/src/components/common/mapFSM/MapStateMachineContext.tsx

```diff
@@ -255,6 +264,26 @@ export const MapStateMachineProvider: React.FC<React.PropsWithChildren<unknown>>
     serviceSend({ type: 'FINISH_SELECTION' });
   }, [serviceSend]);
 
+  const startReposition = useCallback(
+    (
+      repositioningFeatureDataset: LocationFeatureDataset | null,
+      index: number | null,
```

Comment:
Can these be null if they are necessary for the state?

Reply:
good catch - will change